specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/examples/KustoPoolDataConnectionsCreateOrUpdate.json

```diff
@@ -0,0 +1,60 @@
+    "resourceGroupName": "kustorptest",
+    "workspaceName": "synapseWorkspaceName",
+    "kustoPoolName": "kustoclusterrptest4",
+    "databaseName": "KustoDatabase8",
+    "api-version": "2021-04-01-preview",
+    "dataConnectionName": "DataConnections8",
+    "parameters": {
+      "location": "westus",
+      "kind": "EventHub",
+      "properties": {
+        "eventHubResourceId": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.EventHub/namespaces/eventhubTestns1/eventhubs/eventhubTest1",
+        "consumerGroup": "testConsumerGroup1"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Synapse/workspaces/synapseWorkspaceName/kustopools/KustoClusterRPTest4/Databases/KustoDatabase8/DataConnections/DataConnections8",
+        "name": "KustoClusterRPTest4/KustoDatabase8/DataConnections8",
+        "type": "Microsoft.Kusto/Clusters/Databases/DataConnections",
+        "location": "westus",
+        "kind": "EventHub",
+        "properties": {
+          "eventHubResourceId": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.EventHub/namespaces/eventhubTestns1/eventhubs/eventhubTest1",
+          "consumerGroup": "testConsumerGroup1"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Synapse/workspaces/synapseWorkspaceName/kustopools/KustoClusterRPTest4/Databases/KustoDatabase8/DataConnections/DataConnections8",
+        "name": "KustoClusterRPTest4/KustoDatabase8/DataConnections8",
+        "type": "Microsoft.Kusto/Clusters/Databases/DataConnections",
+        "location": "westus",
+        "kind": "EventHub",
+        "properties": {
+          "eventHubResourceId": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.EventHub/namespaces/eventhubTestns1/eventhubs/eventhubTest1",
+          "consumerGroup": "testConsumerGroup1"
+        }
+      }
+    },
+    "202": {
+      "body": {
+        "id": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.Synapse/workspaces/synapseWorkspaceName/kustopools/KustoClusterRPTest4/Databases/KustoDatabase8/DataConnections/DataConnections8",
+        "name": "KustoClusterRPTest4/KustoDatabase8/DataConnections8",
+        "type": "Microsoft.Kusto/Clusters/Databases/DataConnections",
+        "location": "westus",
+        "kind": "EventHub",
+        "properties": {
+          "eventHubResourceId": "/subscriptions/12345678-1234-1234-1234-123456789098/resourceGroups/kustorptest/providers/Microsoft.EventHub/namespaces/eventhubTestns1/eventhubs/eventhubTest1",
+          "consumerGroup": "testConsumerGroup1"
+        }
+      }
```

Comment:
202 response shouldn't have a response body. This comment applies to all examples.

Reply:
It follows the same convention of the original Kusto resources in swagger on which the response presents the creating/updating object to the client.